### PR TITLE
Testing functionality of add and remove functions of the To-do-list APP

### PR DESCRIPTION
**Pull request by Kaweesi-Matia**
## Below is the description of what is included in the pull request

- [x] Add jest framework
- [ ] create babel.config file
- [ ] create the test file and import add and remove functions
- [ ] use describe a method to combine both functions
- [ ] solve linter errors

### CODE REVIEW

File: src/module/addRemoveEdit.test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    expect(addTask).not.toBeNull();
+  });
+  test('remove a task task', () => {
+    // eslint-disable-next-line no-unused-expressions
+    expect(removeTask).not.toBeDefined;
+  });
+  test('remove a task task', () => {
+    expect(removeTask.parentElement).not.toBe('');
+  });
+});
```
Comment (TedLivist) [inline src/module/addRemoveEdit.test.js]: - [ ] Your tests seem not to be properly written. Here you are just testing if the `addTask` and `removeTask` methods simply exist. But that is not the reason for writing these tests. Kindly refactor your tests to demonstrate that `addTask` actually adds a task to a collection/page; and that the `removeTask` method actually removes a task from a collection/page.

- [ ] Remember to also mock the localStorage or DOM and ensure the add and remove functions work as expected on them.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Kaweesi-Matia and @nunegabriel,

Good job so far!
✔️ Good documentation
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kaweesi-Matia** [comment]
- [ ] refactor the code such that the addTask method add a task to the page
- [ ]  refactor the code such that removeTask method remove a task from the page
- [ ] Mock local storage and Dom to ensure that add and remove functions work properly

**Review by dasileker — APPROVED** [review]
### **Hi @Kaweesi-Matia** 

### **Highlight**

- Nice work on adding the required changes.

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**